lib/ts/BaseLogFile.h

```diff
@@ -22,11 +22,11 @@
 
  */
 
-#ifndef BASE_LOG_FILE_H
-#define BASE_LOG_FILE_H
+#pragma once
```

Comment:
I was under the impression `#pragma once` was a drop in replacement for include guards. I'll spend some time today to come up with a fancy regex to use `#pragma once` across the code base.